src/fold.ts

```diff
@@ -369,7 +369,7 @@ class FoldMarker extends GutterMarker {
   eq(other: FoldMarker) { return this.config == other.config && this.open == other.open }
 
   toDOM(view: EditorView) {
-    if (this.config.markerDOM) return this.config.markerDOM(this.open)
+    if (this.config.markerDOM) return this.config.markerDOM(this.open, view)
```

Comment:
I think I would otherwise put view as the first parameter (as it is in most other functions), but I don't want to introduce a breaking change just for this.